SignalMessaging/environment/Environment.m

```diff
@@ -37,7 +38,7 @@ + (Environment *)current
 
 + (void)setCurrent:(Environment *)environment
 {
-    OWSAssert(!sharedEnvironment);
```

Comment:
Can you leave a comment similar to the one in AppContext.m about why we (somewhat surprisingly) expect to set the environment multiple times in the context of SAE?

Reply:
Done.